go-runtime-builder/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
```

Comment:
Other scripts in this repo don't have # on empty lines except go-build.sh. I remove the #'s from go-build.sh instead.